cmd/root_test.go

```diff
@@ -56,24 +65,24 @@ func Test_runRoot(t *testing.T) {
 		args    args
 		wantErr bool
 	}{
-		{"1", args{&cobra.Command{}, []string{"http://test", "/mr/mzxyptlk"}, ""}, true},
-		{"2", args{&cobra.Command{}, []string{"http://127.0.0.1:1793", "/tmp/helm-mirror/test"}, ""}, false},
-		{"3", args{&cobra.Command{}, []string{"%", "/tmp/helm-mirror/test"}, ""}, true},
-		{"4", args{&cobra.Command{}, []string{"http://test", "/tmp/helm-mirror/test"}, "%"}, true},
-		{"5", args{&cobra.Command{}, []string{"http://test", "/tmp/helm-mirror/test"}, "ftp://test"}, true},
-		{"6", args{&cobra.Command{}, []string{"http://127.0.0.1:1793", "/tmp/helm-mirror/test"}, "https://test/com/charts"}, false},
-		{"7", args{&cobra.Command{}, []string{"http://127.0.0.1:1111", "/tmp/helm-mirror/test"}, "https://test/com/charts"}, true},
+		{"1", args{&cobra.Command{}, []string{"http://test", path.Join("/mr", "mzxyptlk")}, ""}, true},
```

Comment:
Same here

Reply:
This one is on purpose, I'm expecting an error while creating the directory when the user has no permissions, should I use another path maybe?

Reply:
You can keep these paths, `path.Join` doesn't care if the path exists or not, it should be:
```golang
path.Join("mr", "mzxyptlk")
```

Reply:
The only problem with that is that the complete path the would be `/path/to/work/dir/mr/mzxyptlk` and that would no fail due permissions.

Reply:
I see, let's keep it this way.